3-UsingAPIs/Week2/homework/ex3-rollAnAce.js

```diff
@@ -10,24 +10,23 @@
 // ! Do not change or remove the next two lines
 const rollDice = require('../../helpers/pokerDiceRoller');
 
-function rollDiceUntil(wantedValue) {
+async function rollDiceUntil(wantedValue) {
   // TODO: rewrite this function using async/await
-  const recurse = () => {
-    return rollDice().then((value) => {
-      if (value !== wantedValue) {
-        return recurse();
-      }
-      return value;
-    });
-  };
-  return recurse();
+  let value;
+  while (value !== wantedValue) {
+    value = await rollDice();
+  }
+  return value;
```

Comment:
Well done, much easier than using Promises, right?

Reply:
Yes , that's right 100%